exonum-java-binding/core/src/main/java/com/exonum/binding/core/transaction/TransactionMethod.java

```diff
@@ -31,9 +31,9 @@
  *   <li>be public
  *   <li>have exactly two parameters - the
```

Comment:
This sentence has become too complex, I suggest rewriting as the following: 
"\<p>The method should be {@code public} and have the following parameters: [as a list] 1. ... 2. ..."